resources/integration_test/tests/queries.yaml

```diff
@@ -227,6 +227,27 @@ tests:
           # 11: {"$undefined":true},
       ]
 
+  - description: query_types_other using simple_types_only mode
+    test_definition: SELECT * FROM types_other
+    db: integration_test
+    simple_type: true
+    expected_result:
```

Comment:
I didn't change the expected results at all, and this test worked. Is this how it should be in simple_types_only mode?

Reply:
This is because you don't have any metadata test.
You need `expected_sql_type`, `expected_precision, `expected_display_size` and maybe `expected_length` with the arrays set with the correct values.
I would add a test just for checking metadata in simple mode:
`query_types_other using simple_types_only_mode` (no need to check the expected result since we ask for the string representation in both cases) and maybe the same query as `select_star_unordered`.
Or even better, find a query which returns columns with complex and simple types and add only the metadata tests in both simple and standard mode. Making sure that in simple mode the complex types are returned as VARCHAR and the other types stay the same then for the standard mode.